plugins/engines/ospray/ispc/render/ParticleRenderer.ispc

```diff
@@ -20,154 +18,142 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-// Brayns
 #include <plugins/engines/ospray/ispc/render/utils/AbstractRenderer.ih>
 
-const float NEST_TIMESTEP = 0.1f;
-
 struct ParticleRenderer
 {
     AbstractRenderer abstract;
 
-    int randomNumber;
+    // Transfer function / Color map attributes
+    uniform vec4f* uniform colorMap;
+    uniform vec3f* uniform emissionIntensitiesMap;
+    uint32 colorMapSize;
+    float colorMapMinValue;
+    float colorMapRange;
 
+    // Simulation data
     uniform float* uniform simulationData;
     int64 simulationDataSize;
-    uniform vec4f* uniform transferFunction;
-    uniform float* uniform transferFunctionEmissionData;
-    uint32 transferFunctionSize;
+
+    // Alpha correction
+    float alphaCorrection;
 };
 
 inline varying vec4f
```

Comment:
A similar function is used also for the simulation renderer, can they be merged?